Engine/source/gui/controls/guiMLTextCtrl.cpp

```diff
@@ -669,11 +669,11 @@ void GuiMLTextCtrl::getCursorPositionAndColor(Point2I &cursorTop, Point2I &curso
 {
    S32 x = 0;
    S32 y = 0;
-   S32 height = mProfile->mFont->getHeight();
+   S32 height = mProfile && mProfile->mFont ? mProfile->mFont->getHeight() : 0;
```

Comment:
This needs to be the following as parenthesis is very important to avoid turning `height` into a true/false evaluation result:
```C++
S32 height = (mProfile && mProfile->mFont) ? mProfile->mFont->getHeight() : 0;
```